assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -16,23 +16,43 @@
  * limitations under the License.
  */
 
+/**
```

Comment:
Usually we put WordPress dependencies underneath external ones, and above "internal" ones.